recursion/intro-to-recursion.md

```diff
@@ -0,0 +1,114 @@
+# Intro to Recursion
+
+## Learning Goals
+
+- Define recursion
+
+## Introduction
+
+A powerful problem-solving technique in computer science is to use recursion. It takes advantage of operations that computers excel at.
+
+However, recursion happens around us all the time! Observing and naming recursion is a great way to begin thinking about recursion.
```

Comment:
basically the same sentence, but reworked a bit to flow better from the previous change.
```suggestion
But recursion applies to more than just problems. Recursion happens around us all the time. Observing and naming recursion in the real world is a great way to begin thinking about recursion in our programs!
```